### DISCUSSION TITLE
How do i use the setup and use the Dashboard

### DISCUSSION THREAD

**Discussion by Forrezt-Blank** [dimaguy/ImpostorHQ#20] [Q&A] [answered]
Hi, i downloaded the latest realease & made port 22024 since 22023 already exists and both ports have udp and tcp enabled & firewall is fine so i opened the dashboard client.hmtl and i put in my API and then click login and nothing happens, am doing something wrong? caould you walk me throw the steps (if need be i do own my own domain online)

**Comment by dimaguy** [comment, marked as answer]
Hi @Forrezt-Blank 
There is no need to change port configurations since Impostor only uses the UDP transport, leaving TCP free for ImpostorHQ.
You also do not need your own domain.
I am going to need the following things:

- Firewall rules and port forwarding rules applied
- Browser's Console logs
- Impostor's Console output from beginning to your attempt of logging in
